### DISCUSSION TITLE
New track elements are all the same price

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#19920] [Feature requests]
All the diagonal slope and banked track elements cost the same amount of money - 294912. I believe this was a temporary cost value since the flyer loops cost this and they're directly above the new track pieces.

These track piece prices should be balanced based on their size, using other track elements as a guide.

**Comment by X123M3-256** [comment]
Yes, almost all the track data is filled with dummy data. I forgot to fill in real values before submitting the PR and there are a number of bugs resulting from this. I need to get around to fixing it.

**Comment by karst** [comment]
has this been worked on?

**Reply by spacek531** [reply]
I believe this is done.

**Reply by karst** [reply]
So I thought. Then we can close this

**Reply by X123M3-256** [reply]
Fixed in #20687
